helm/charts/determined/templates/master-config.yaml

```diff
@@ -122,6 +122,9 @@ data:
       fluent:
         {{- toYaml .Values.fluent | nindent 8}}
       {{- end }}
+      {{- if .Values._reattachResources }}
+      _reattach_resources: {{ .Values._reattachResources }}
```

Comment:
we should have a plan for flipping this on in all cases soon

Reply:
I agree, should be default on in all resources managers eventually